vendor.conf

```diff
@@ -37,7 +37,7 @@ github.com/mitchellh/hashstructure 2bca23e0e452137f789efbc8610126fd8b94f73b
 #get libnetwork packages
 
 # When updating, also update LIBNETWORK_COMMIT in hack/dockerfile/install/proxy accordingly
-github.com/docker/libnetwork f30a35b091cc2a431ef9856c75c343f75bb5f2e2
+github.com/docker/libnetwork a79d3687931697244b8e03485bf7b2042f8ec6b6
```

Comment:
I would suggest updating to the latest which is cfa742c8abe6f8e922f6e4e920153c408e7d9c3b